client-ui/src/components/pages/auth/login/Login.tsx

```diff
@@ -0,0 +1,132 @@
+import "./login.css";
+import { Form, Input, Checkbox, Button, Row, Col } from "antd";
+import Logo from "../../../../assets/Logo/Logo.png";
+import Image from "../../../../assets/Image/Login.png";
+
+function Login() {
+  return (
+    <div>
+      <div className="container">
+        <div className="purplecard">
+          <div className="innercontent">
+            <div className="logo">
+              <img src={Logo} alt="Company Logo" />
+            </div>
+            <div className="image">
+              <img src={Image} alt="Login Image" />
+            </div>
+          </div>
+        </div>
+        <div className="whitecard">
+          <div className="roundtitle">Welcome Back</div>
+          <div className="innercontent">
+            <div className="head">
+              <h1>Login your account</h1>
+            </div>
+            <Form layout="vertical">
+              <p>Username or Email</p>
+              <Input bordered={false} />
+            </Form>
+            <Form layout="vertical">
+              <p>Password</p>
+              <Input bordered={false} />
+            </Form>
+            <Form>
+              <Form.Item
+                name="remember"
+                valuePropName="checked"
+                wrapperCol={{ offset: 0, span: 12 }}
+              >
+                <Checkbox>Remember me</Checkbox>
+              </Form.Item>
+              <Form.Item wrapperCol={{ offset: 6, span: 16 }}>
+                <Button type="primary">Login</Button>
+              </Form.Item>
+            </Form>
+            <Form>
+              <Row>
+                <Col span={12}>
+                  {" "}
+                  <a href="#">Forget Password?</a>
```

Comment:
For placeholder URL use `/` or `String("")` don't use `#` or empty value.
```suggestion
                  <a href="/">Forget Password?</a>
```